theano/tensor/tests/test_subtensor.py

```diff
@@ -97,18 +98,17 @@ def setUp(self):
         Subtensor.debug = False
         utt.seed_rng()
 
-    def eval_output_and_check(self, t, list=False, mode=None):
+    def eval_output_and_check(self, t, op_type=None, mode=None, length=1):
```

Comment:
Also, you can't use `self` in the definition of a default parameter because `self` isn't bound to anything yet.  Default parameters are evaluated at function definition time — not execution time.